.github/dependabot.yml

```diff
@@ -14,5 +14,13 @@ updates:
   versioning-strategy: lockfile-only
   labels:
   - dependabot-deps-update
+- package-ecosystem: pip
+  directory: /docs
+  schedule:
+    interval: daily
```

Comment:
I usually dislike frequent updates because they annoy people with notifications for no reason. This is especially painful, on low-traffic projects...